js-exercises/animal-sort/animalSort.js

```diff
@@ -0,0 +1,15 @@
+function animalSort(arr) {
```

Comment:
We are taking input as an array of animals. `animals` would have been a better name here instead of plain old `args`.